elections/steering/2024/election.yaml

```diff
@@ -0,0 +1,20 @@
+name: 2024 Steering Committee Election
+organization: Kubernetes
+# Start of day in UTC for opening: 2023-08-29 00:00:01
+start_datetime: 2024-08-27 00:00:01
+# End of day Anywhere on Earth for closing. Write 2023-09-26 as: 2023-09-27 11:59:59
+end_datetime: 2024-09-27 11:59:59
+no_winners: 3
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - employer
+  - slack
+election_officers:
+  - bridgetkromhout
+  - cblecker
+  - Priyankasaggu11929
+eligibility: Kubernetes Org members with 50 or more contributions in the last year can vote.  See [the election guide](https://github.com/kubernetes/community/tree/master/elections/steering/2024)
+exception_description: Not all contributions are measured by DevStats.  If you have contributions that are not so measured, then please request an exception to allow you to vote via the Elekto application.
```

Comment:
We should link to the [Voter exception](https://github.com/kubernetes/community/blob/e9fe9e62167380984b3b22c88ca7e174d0be3cc3/elections/steering/2024/README.md#voter-exception) section in `exception_description`?

That will help pointing to:
(i) how to file a voter exception request
(ii) point what contributions might or might not be considered for voter exception request

Reply:
Sure, we could add a link, but this is the yaml file that defines the election config, so it's not human-read usually. I am not 100% sure how Elekto would consume it if we had links in the text fields - @jberkus may know?

Reply:
Just checked the docs here 
https://elekto.dev/docs/administration/creating/#electionyaml-variables

```
exception_description: Text to display before the Request Exception link. Optional, changeable at any time.
```

^ which means the "how to file a voter exception request" is taken care already.

Reply:
> how Elekto would consume it if we had links in the text fields

I took reference from the `eligibility` field. That has a link to the `the election guide` doc.

But not sure whether links render or not.

Reply:
The UI already displays a link to the exception form.

Reply:
Also, it's important to keep the text that actually displays on the UI relatively short.  For folks who want the full details, that's what the election README page is for.